src/transport/net_ib.cc

```diff
@@ -92,7 +102,14 @@ static void* ncclIbAsyncThreadMain(void* args) {
     char *str;
     if (ncclSuccess != wrap_ibv_event_type_str(&str, event.event_type)) { break; }
     if (event.event_type != IBV_EVENT_COMM_EST)
-      WARN("NET/IB : Got async event : %s", str);
+    {
+      for (int d=0; d<ncclNIbDevs; d++) {
+        if (strcmp(context->device->name, ncclIbDevs[d].devName) == 0 && ncclParamResilientEnabled())
+        {
+          WARN("NET/IB : Got async event : %s event type: %d deviceid:%d device name:%s", str, event.event_type, d, context->device->name);
+        }
+      }
```

Comment:
shall we keep the original code and print the device name for all the warnings instead of the resilient enabled ones?